packages/core/src/StateMachine.ts

```diff
@@ -338,15 +348,20 @@ export class StateMachine<
   /**
    * Returns the initial `State` instance, with reference to `self` as an `ActorRef`.
    */
-  public getInitialState(): State<TContext, TEvent, TResolvedTypesMeta> {
-    const { preInitialState } = this;
-    const nextState = resolveMicroTransition(this, [], preInitialState);
+  public getInitialState(
```

Comment:
I don't think this is needed on the interpreter - at least not as a public method. The consumer should only care about `.getSnapshot()`